src/unity/python/turicreate/data_structures/image.py

```diff
@@ -194,6 +194,10 @@ def _to_pil_image(self):
         from PIL import Image as _PIL_Image
         return _PIL_Image.fromarray(self.pixel_data)
 
+    def save(self, filename):
+        from PIL import Image as _PIL_Image
```

Comment:
Same comment as last time - please use `_to_pil_image` from within this function, so we are sharing code wherever possible (i.e. in the future, the way we convert to a PIL image may change, and if we have to change it in two places the logic may diverge).

Reply:
Yep, thanks! Done.